src/Controllers/StudentsController.cs

```diff
@@ -216,6 +217,14 @@ public async Task<JsonResult>__classrooms___(string accessCode, string sid)
                         {
                             return Json("codeinvalid");
                         }
+                        
+                        //return Json(student);
+                        if (student.CollegeId == null || student.CollegeId == " " ||
+                            student.HscBatch == null || student.HscBatch == " " ||
+                            student.Shift == null || student.Shift == " ")
+                        {
+                            return Json("NeedCompleteStudentProfile");
+                        }
```

Comment:
What if the  student.CollegeId = "        " (more than single space) ?
Something like **Type Casting + Trim + string.IsNullOrEmpty** combination can be perfect.
This test-case needs to be re-optimized in future. Add this in trello. 

For now, it's **all right** 👍👍👍